web/assets/data-loss-warning.js

```diff
@@ -0,0 +1,130 @@
+export class DataLossWarning {
+    constructor(saveOrReturnComponent) {
```

Comment:
I feel like we may as well just query for this in here since we are grabbing other stuff from `document`. And maybe use the tasklist `<a>` instead of the button group, because we don't do anything with the "save" bit

Reply:
Yeah fair point - fell into the pattern of existing components we have.